src/main/java/teammates/ui/controller/InstructorFeedbackCopyAction.java

```diff
@@ -36,13 +38,13 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         
         InstructorFeedbacksPageData data = new InstructorFeedbacksPageData(account);
         
+        FeedbackSessionAttributes fs = null;
         try {
-            FeedbackSessionAttributes fs = logic.copyFeedbackSession(copiedFeedbackSessionName,
-                                                                     copiedCourseId,
-                                                                     feedbackSessionName,
-                                                                     courseId,
-                                                                     instructor.email);
-            data.newFeedbackSession = fs;
+            fs = logic.copyFeedbackSession(copiedFeedbackSessionName,
+                                             copiedCourseId,
+                                             feedbackSessionName,
+                                             courseId,
+                                             instructor.email);
```

Comment:
Indentation is off here; when there are a lot of parameters, it may be better to put multiple variables on each line instead.